config/secrets.yml

```diff
@@ -0,0 +1,10 @@
+development:
+  secret_key_base: 8b16915e35590d84c65509821a16dd29561866c937e46d0cce6be9e815b60a0cea81ad96ed5800d56a5de9b31f9bf0a32a45d7435e7ffd91aa52bb185ce64555
+
+test:
+  secret_key_base: 53717d6a1d28dc695039f211666e221263891e60e5a19259c22d025f4a601c8ac3b3ed16ec5878863731adbe1dc6f7f1f4b5b4d006b79e19847d5ca44a2eb708
+
+# Do not keep production secrets in the repository,
+# instead read values from the environment.
+production:
```

Comment:
How do we implement this for production? Rails 4.2 apparently requires these secret keys now? Seems weird we have to do this for a gem, though. For the development and test secrets I just did:
```
$ irb
>> require 'securerandom'
=> true
>> SecureRandom.hex(64)
=> "3fe397575565365108556c3e5549f139e8078a8ec8fd2675a83de96289b30550a266ac04488d7086322efbe573738e7b3ae005b2e3d9afd718aa337fa5e329cf"
>> exit
```